src/components/flex.js

```diff
@@ -47,6 +47,15 @@ export const Box = styled.div`
   box-sizing: ${(props) => props.boxSizing ?? "content-box"};
 
   position: ${(props) => props.position ?? "static"};
+
+  &::-webkit-scrollbar {
```

Comment:
it's not a Flex component styles, it is a global styles, move it into global object